### ISSUE TITLE
Old graceful-fs version support will be removed soon

### ISSUE THREAD

**Issue by ChALkeR** [twigjs/twig.js#399]
Hi. `twig` is currently `#7` in my [list of modules](https://gist.github.com/ChALkeR/d39b49642e02277d2292d66c2ea0bced) that would likely to be broken when https://github.com/nodejs/node/pull/6413 lands (which is anticipated to happen before v7.0).

Deps tree:

```
└─┬ twig@0.9.5
  └─┬ phpjs@1.3.2
    ├─┬ glob@3.2.1
    │ └── graceful-fs@1.2.3 
    └─┬ mocha@1.17.0
      └─┬ glob@3.2.3
        └── graceful-fs@2.0.3 
```

Note that `graceful-fs@1` and `graceful-fs@4` are ok and would not be broken by this, but a `require()` on `graceful-fs@2` or `graceful-fs@3` would cause an error soon.

If `mocha` is used in runtime, you should somehow update your deps to use a newer `mocha` version (which is already fixed) or avoid using it at all. Note that a deprecation warning upon installation would happen anyway, even if `mocha` isn't used at all — there is no way to determine that at the installation time.

Tracking: https://github.com/nodejs/node/issues/5213.

**Comment by ChALkeR** [comment]
Note that [phpjs](https://www.npmjs.com/package/phpjs) was superseded by [locutus](https://www.npmjs.com/package/locutus), it seems. Current version of `locutus` is fine, and it doesn't use broken `graceful-fs` versions.

**Comment by ChALkeR** [comment]
This looks to be fixed by #376, but no new releases happened since then. Could you publish a new release so I could exclude twig from my list, please? =)

**Comment by dave-irvine** [comment]
Latest release published.

https://snyk.io/test/github/twigjs/twig.js
